src/battle-animations.ts

```diff
@@ -1804,6 +1804,7 @@ class PokemonSprite extends Sprite {
 		laserfocus: ['Laser Focus', 'good'],
 		spotlight: ['Spotlight', 'neutral'],
 		itemremoved: null,
+		furycutter: null,
```

Comment:
This assignment doesn't seem to do anything, the furycutter volatile still appears as some default effect.

Interestingly, I tried removing L1806 (since itemremoved is the volatile I am trying to mimic), but that did not produce a default effect for itemremoved
![furycutter](https://user-images.githubusercontent.com/29032443/117309392-fddd9380-ae47-11eb-99ff-96a8843842d8.png)
. It again did nothing.

Reply:
Yes, `battle-animations` is not built by default because it bundles `battle-animations-moves` which makes the build extremely slow.

Simply `./build full` to opt into building it, and it should work after that.

Reply:
I think Fury Cutter _should_ show up on the bar, though. As should Echoed Voice.

Reply:
(With its current multiplier, correct for the generation. Although feel free to leave that for a future PR.)